### DISCUSSION TITLE
Better and clearer documentation for RustDesk OpenSource Server.

### DISCUSSION THREAD

**Discussion by filu34** [rustdesk/rustdesk-server#377] [General]
Right now whole documentation is a mess, and lacks many crucial informations. I did all steps from there and strill can't result in properly working hbbs server, that connects with a clients. 

Make better documentation.

**Comment by soekmego** [comment]
To add some constructive points, especially for the pro version since we are going to use rustdesk commercially and already have orders on the table:

**Env vars**
I followed the documentation on the [website for the pro version](https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/installscript/docker/) yet could not find any traces of possible env vars until I opened up [GitHub](https://gitlab.digimagical.com/images/containers/rustdesk-server-pro#env-variables). Having the env var matrix available in the docs would have saved me a little bit of time. Default values would have been nice too.

**Ports**
[On the OSS install page](https://rustdesk.com/docs/en/self-host/rustdesk-server-oss/docker/) the ports are listed nicely (even 21114 which is not required for the OSS version), on the Pro version page they are missing. 

**Search**
I think the documentation page is lacking a search function.

I cant think of anything else right now but will add new comments as soon as I stumble upon something.
